### PR TITLE
add readme to plugins

### PR DESCRIPTION
**Pull request by oxarbitrage**
Added a table to plugins in a BSIP style to keep control of current plugins with some basic information for visitors landing in this section of our software.

### CODE REVIEW

File: libraries/plugins/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+The bitshares plugins are a collection of tools that brings new functionality without the need of modifications in the consensus and more sensitive areas of the bitshares-core.
+
+The main source of I/O of the bitshares blockchain is the API. Plugins are a more powerful alternative to build more complex developments for when the current API is not enough.
+
+Plugins are optional to run by node operator according to their needs however all plugins here will be compiled. There are plans for optional build of plugins at: #533 
+
+# Available Plugins
+
+Number              | Name                     | Description                                                                 | Category       | Status
```
Comment (pmconrad) [inline libraries/plugins/README.md]: Please add column with database space_id.

File: libraries/plugins/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# BitShares Plugins
+
+The bitshares plugins are a collection of tools that brings new functionality without the need of modifications in the consensus and more sensitive areas of the bitshares-core.
+
+The main source of I/O of the bitshares blockchain is the API. Plugins are a more powerful alternative to build more complex developments for when the current API is not enough.
+
+Plugins are optional to run by node operator according to their needs however all plugins here will be compiled. There are plans for optional build of plugins at: #533
```
Comment (pmconrad) [inline libraries/plugins/README.md]: Please use proper github url instead of hashtag reference.

**Comment by pmconrad** [comment]
Thanks. Does the "number" column have a particular meaning? There is no useful ordering of plugins within the code, perhaps remove the column and order plugins alphabetically?

File: libraries/plugins/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# BitShares Plugins
+
+The bitshares plugins are a collection of tools that brings new functionality without the need of modifications in the consensus and more sensitive areas of the bitshares-core.
+
+The main source of I/O of the bitshares blockchain is the API. Plugins are a more powerful alternative to build more complex developments for when the current API is not enough.
+
+Plugins are optional to run by node operator according to their needs however all plugins here will be compiled. There are plans for optional build of plugins at: [Issue 533](https://github.com/bitshares/bitshares-core/issues/533)
```
Comment (jmjatlanta) [inline libraries/plugins/README.md]: ... to their needs. However, ...

**Pull request merged**